src/mongo-client.js

```diff
@@ -14,7 +14,12 @@ class MongoClient {
         } else {
           try {
             const collection = client.db().collection(parameters.collection);
-            const cursor = collection.find(parameters.find || {}, parameters.options || {});
+            let cursor = null;
+            if (parameters.aggregate !== undefined) {
```

Comment:
Maybe change to `if (parameters.aggregate) {` ?

Reply:
Done.